modules/_labs/panel/react/lib/PanelHeader.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {colors, spacing, H4, CanvasColor} from '@workday/canvas-kit-react-core';
+import {IconButton} from '@workday/canvas-kit-react-button';
+import {CanvasSystemIcon} from '@workday/design-assets-types';
+import {xIcon} from '@workday/canvas-system-icons-web';
+
+export interface PanelProps extends React.HTMLAttributes<HTMLDivElement> {
+  headerTitle?: string;
+  handleClose?: () => void;
+  iconLabel: string;
+  icon: CanvasSystemIcon;
+  headerColor: CanvasColor | string;
+}
+
+const headerHeight = 56;
+
+const HeaderContainer = styled('div')<Pick<PanelProps, 'headerColor'>>(
+  {
+    height: headerHeight,
+    display: 'flex',
+    alignItems: 'center',
+    padding: `0 ${spacing.s}`,
+    justifyContent: 'space-between',
+    borderBottom: `1px solid ${colors.soap500}`,
```

Comment:
Design thing, but this border may look weird with some colors.

Reply:
I'll talk to Michael about this and consider a different approach!